src/main/java/org/apereo/portlet/soffit/model/v1_0/Definition.java

```diff
@@ -26,16 +26,21 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * Provides information about the publication record of the soffit within the
+ * portal.
+ *
+ * @author drewwills
+ */
 public class Definition {
 
     private String title;
-    private String name;
     private String fname;
     private String description;
-    private int timeout;
 
     private Set<String> categories = new HashSet<>();
     private Map<String,List<String>> parameters = new HashMap<>();
+    private Map<String,List<String>> preferences = new HashMap<>();
```

Comment:
Moved here from the `Request` object.  The way we're using them (i.e. admin config), preferences are logically a part of the `Definition`.